repositories.bzl

```diff
@@ -13,351 +13,341 @@
         version = "v0.3.1",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_census_instrumentation_opencensus_proto",
         importpath = "github.com/census-instrumentation/opencensus-proto",
         sum = "h1:glEXhBS5PSLLv4IXzLA5yPRVX4bilULVyxxbrfOtDAk=",
         version = "v0.2.1",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_client9_misspell",
         importpath = "github.com/client9/misspell",
         sum = "h1:ta993UF76GwbvJcIo3Y68y/M3WxlpEHPWIGDkJYwzJI=",
         version = "v0.3.4",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_envoyproxy_go_control_plane",
         importpath = "github.com/envoyproxy/go-control-plane",
         sum = "h1:4cmBvAEBNJaGARUEs3/suWRyfyBfhf7I60WBZq+bv2w=",
         version = "v0.9.1-0.20191026205805-5f8ba28d4473",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_envoyproxy_protoc_gen_validate",
         importpath = "github.com/envoyproxy/protoc-gen-validate",
         sum = "h1:EQciDnbrYxy13PgWoY8AqoxGiPrpgBZ1R8UNe3ddc+A=",
         version = "v0.1.0",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_golang_glog",
         importpath = "github.com/golang/glog",
         sum = "h1:VKtxabqXZkF25pY9ekfRL6a582T4P37/31XEstQ5p58=",
         version = "v0.0.0-20160126235308-23def4e6c14b",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_golang_mock",
         importpath = "github.com/golang/mock",
         sum = "h1:G5FRp8JnTd7RQH5kemVNlMeyXQAztQ3mOWV95KxsXH8=",
         version = "v1.1.1",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_golang_protobuf",
         importpath = "github.com/golang/protobuf",
         sum = "h1:+Z5KGCizgyZCbGh1KZqA0fcLLkwbsjIzS4aV2v7wJX0=",
         version = "v1.4.2",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_google_go_cmp",
         importpath = "github.com/google/go-cmp",
         sum = "h1:X2ev0eStA3AbceY54o37/0PQ/UWqKEiiO2dKL5OPaFM=",
         version = "v0.5.2",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_google_renameio",
         importpath = "github.com/google/renameio",
         sum = "h1:GOZbcHa3HfsPKPlmyPyN2KEohoMXOhdMbHrvbpl2QaA=",
         version = "v0.1.0",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_gordonklaus_ineffassign",
         importpath = "github.com/gordonklaus/ineffassign",
         sum = "h1:vc7Dmrk4JwS0ZPS6WZvWlwDflgDTA26jItmbSj83nug=",
         version = "v0.0.0-20200309095847-7953dde2c7bf",
     )
-    _maybe(
-        go_repository,
+    go_repository(
         name = "com_github_jhump_protoreflect",
+        # Manually added in order to disable testproto BUILD file generation.
```

Comment:
just a qq - is this file (or at least the content of this file) generated? If yes, how do we make sure that this "disable" parameter does not get overridden?

Reply:
It is retained by `gazelle`. I've done some tests and it seems to be kept. My guess is because `gazelle` does some merging of the existing rules with `buildozer`. I also updated the comment to remove the implication that I manually update that attribute every time - I've done it once, and `gazelle` keeps it now :)

Reply:
And yes, the contents of the _macro_ is generated - except for the `gapic-generator` dependency, that was added by me, but is retained by `gazelle`.